### ISSUE TITLE
Use Templating in index.html

### ISSUE THREAD

**Issue by ethanchewy** [ethanchewy/PythonBuddy#36]
http://flask.pocoo.org/docs/1.0/tutorial/templates/

**Comment by dgmouris** [comment]
Really cool project. Huge respect!

What do you mean by templating, do you want to separate out a base.html? 

I'd love to help out as I think this project is really neat. Thanks for your time.

**Comment by ethanchewy** [comment]
Awesome! Thanks for taking a look at this project.

I was thinking about using Jinja for templating: http://flask.palletsprojects.com/en/1.1.x/tutorial/templates/

Hasn't been necessary in the past since we only have really one HTML file to work with but I want to support in the future perhaps the possibility of users being able to sign in, log out, and view their previous code submissions which would be require templating. 

So, this step is mainly converting the raw HTML into a templating system using Jinja to future proof the project when it expands!

For example, we could convert the current index.html into parts like: header, examples, sponsor, text editor, etc.

**Comment by dgmouris** [comment]
Thanks for the really quick response, that's super nice and awesome.

Jinja2 would be a great idea, especially if you're planning project expansion. I'd love to help out doing this if you don't mind.

Let me know if that would be alright with you if I worked on this.

**Comment by ethanchewy** [comment]
Yes, go ahead. Feel free to add any questions, comments, or suggestions in this thread throughout the process!

**Comment by dgmouris** [comment]
Awsome @ethanchewy thanks for the help.

**Comment by dgmouris** [comment]
Hey @ethanchewy I've added a lot of blocks just because I think this might be handier in the future.

Let me know if you'd like to explain/remove/add more blocks and I'd be really happy to.

I hope you have a great day!
